letsencrypt/colored_logging.py

```diff
@@ -0,0 +1,47 @@
+"""A formatter and StreamHandler for colorizing logging output."""
+import logging
+import sys
+
+from letsencrypt import le_util
+
+
+class StreamHandler(logging.StreamHandler):
+    """Sends colored logging output to a stream.
+
+    If the specified stream is not a tty, the class works like the
+    standard logging.StreamHandler. Default red_level is logging.WARNING.
+
+    :ivar bool colored: True if output should be colored
+    :ivar bool red_level: The level at which to output
+
+    """
+    _RED = '\033[31m'
```

Comment:
This could be moved to `le_util` as well, next to `ANSI_SGR_RESET`.

In fact we could start thinking about using a library for that